cli/src/main/java/com/devonfw/tools/ide/io/FileAccessImpl.java

```diff
@@ -291,27 +294,66 @@ private void copyRecursive(Path source, Path target, FileCopyMode mode) throws I
   }
 
   @Override
-  public void symlink(Path source, Path targetLink) {
+  public void symlink(Path source, Path targetLink, boolean relative) {
+
+    if (!source.isAbsolute()) {
+      throw new IllegalStateException("When creating a symbolic link the source (" + source
+          + ") must be an absolute path. If you want to create a relative link, "
+          + "then pass the source as an absolute path and set the relative flag to true.");
+    }
```

Comment:
The rationale was that I wanted to force the use of the flag "relative" instead of just passing a relative path as source. But I can change it to also work with passing realative sources.